src/docker.cc

```diff
@@ -80,17 +79,18 @@ std::vector<MetadataUpdater::ResourceMetadata>
         if (config_.VerboseLogging()) {
           LOG(INFO) << "Inspect response: " << body(inspect_response);
         }
-        json::value parsed_metadata =
+        json::value raw_docker =
             json::Parser::FromString(body(inspect_response));
         if (config_.VerboseLogging()) {
-          LOG(INFO) << "Parsed metadata: " << *parsed_metadata;
+          LOG(INFO) << "Parsed metadata: " << *raw_docker;
         }
+
         const MonitoredResource resource("docker_container", {
           {"location", zone},
           {"container_id", id},
         });
 
-        const json::Object* container_desc = parsed_metadata->As<json::Object>();
+        const json::Object* container_desc = raw_docker->As<json::Object>();
```

Comment:
I'm not sure why these changes are being bundled in here. This looks like a separate PR.

Reply:
Yes, this is basically drive-by cleanup. I'm happy to move these into the next PR that deals specifically with Docker.
Done.